pkg/mqtrigger/mqtrigger.go

```diff
@@ -40,10 +44,68 @@ func Start(logger *zap.Logger, routerUrl string) error {
 	// Message queue type: nats is the only supported one for now
 	mqType := os.Getenv("MESSAGE_QUEUE_TYPE")
 	mqUrl := os.Getenv("MESSAGE_QUEUE_URL")
+
+	// For authentication with message queue
+	secrets, err := readSecrets(logger)
```

Comment:
Since we support different types of message queue you should do this part at `makeKafkaMessageQueue`.

Reply:
I think moving this part into `makeKafkaMessageQueue` makes sense but I was thinking of secrets here as more of a generic thing because other message queues might need this in the future. Because we read all the secrets and make it available to any message queue, tt might save us some work in the future (say we want to have authentication for NATS or some other message queue in the future) because we would have a pipeline to read secrets ready. 

What do you think?

Reply:
Oops, sorry, I forgot to remove this comment last night. You're right for this.